### PR TITLE
SETUP "HELLO MICROVERSE PROJECT"

### PR DESCRIPTION
**Pull request by Talha-Hanif5666**
In this project, I have added a "Hello Microverse " program that was made with HTML and CSS. I have added linters also to check the errors in the project. I have also included the .gitignore file to limit the uploading of extra files to the GitHub repo.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test.md
```
Comment (ichala) [inline .gitignore:1]: - [ ] good job creating the .gitignore file and adding the `test.md` inside of it 👏 ,  as part of the requirements , kindly list  the `node_module` folder as well

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Talha-Hanif5666 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- .gitignore file is created✔️ 

_**Notes To Consider For Your Next Projects**_

- ⚠️  Good Job so far, your commit message is clear and meaningful but kindly make sure in your next projects, The commit message should be written in the imperative tense and capitalized example: Add styling to the header, Add .gitignore file, etc ...
- ⚠️ Your branch name should be a name of a feature and not literally named `feature`, it should be a bit descriptive and meaningful example: `setup-project`, since changing the name of the branch will cause some issues and one of them is closing opened PR, kindly make sure to implement that in your future projects


 ## Required Changes ♻️

- [ ] 🛑 Linters analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs, Currently, your repo doesn't have any linters configuration, kindly follow [this guide](https://github.com/microverseinc/linters-config/tree/master/html-css) on how to install them and kindly make sure all the tests are green 

- [ ] 🛑 Pull Request should have a descriptive title telling you what changes are inside. kindly update your PR title to something more descriptive, like; ' Setup Hello Project'

- [ ] 🛑 Pull Request should have a summary describing introduced changes in more detail. kindly update your pull request description to describe what changes you made to this PR

- [ ] 🛑 Currently your project doesn't have Readme.md file , to make your project looks professional and well-documented, kindly use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by microverse and try to put all of your project information inside the correct section

- [ ] 🛑 **_Check The Comment Below_**




## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+node_modules/
+.hintrc
+*json
```
Comment (leonardodiasb) [inline .gitignore:4]: - [ ] the `node_modules/` folder was correctly ignored but you should not ignore the `.hintrc` file and the `json` files. The `package.json`  and `package-lock.json` will help the user to install de dependencies of the project and need to be committed. Pleas remove those lines from the `.gitignore` file.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">   
+<title>Trying</title> 
+<link rel="stylesheet" href="E:\Hello-Microverse\Style.css">
```
Comment (leonardodiasb) [inline index.html:6]: - [OPTIONAL] For referencing the links and files in your project you should not use the location of where the file is in your personal computer. If someone clones this repository the location of their `Style.css` may be different and the HTML file will not load, so please change that.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Talha-Hanif5666, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors .✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Talha-Hanif5666) [reply index.html:6]: can you help me in this?

**Review by PinkMoon25 — APPROVED** [review]
Hi @Talha-Hanif5666 , 

Good job completing your firs project :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters are setup correctly :heavy_check_mark: 
Added Html and Css file :heavy_check_mark: 
Added .gitignore file correctly :heavy_check_mark: 
Linters are passong :heavy_check_mark: 

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**